### ISSUE TITLE
Software error - task held when run on A500/KS1.3

### ISSUE THREAD

**Issue by janossuranyi** [weiju/amiga_hardware_in_c#1]
Hello,

I watched your amiga C tutorial video and I just want to try myself. Everything run fine, I can compile your project successfully, but when I try to run on an emulated A500/1.3 Kickstart it just fails to start.
"An error dialog comes up and says: 
Software error - task held
Finish ALL disk activity
Select CANCEL to reset/debug"
But when I've tried on an emulated A1200/3.1 Kickstart it runs perfectly. So it seems that the proble is related to kickstart version.

Output of the make command:
bash-3.2$ make
vc +kick13 -I/opt/vbcc/ndk_3.9/include/include_h -c99 -O2 startup.c -c -o startup.o
vc +kick13 startup.o -lamiga -lauto -o startup
# ls -l
-rw-r--r--  1 johnny  staff   235 Jul 11 21:27 Makefile
-rwxr-xr-x  1 johnny  staff  8792 Jul 11 21:53 startup
-rw-r--r--  1 johnny  staff  1906 Jul 11 21:27 startup.c
-rw-r--r--  1 johnny  staff   844 Jul 11 21:53 startup.o

**Comment by weiju** [comment]
Hi, thanks for reporting the issue. Depending on when you downloaded your version of VBCC, it is possible this is caused by a bug in the VBCC kick13 target which causes programs to crash on systems earlier than Kickstart 2.x. I have experienced this myself when I used version 0.9e of the compiler a few months ago.

It has been fixed in the current version 0.9f of VBCC, so I would recommend installing the latest version of VBCC and see if the problem persists
